### ISSUE TITLE
Full Immersion Rec Studio reveals hosted cards to runner in log

### ISSUE THREAD

**Issue by powerguynz** [mtgred/netrunner#1827]
When hosting a card on Rec Studio it is being revealed to the runner in the logs. I.e. x spends click to use Full Immersion RecStudio to install and host **Corporate Sales Team**. This information shouldn't be getting revealed to the runner - the cards are installed face down and should behave like any other asset until accessed or rezzed.

**Comment by throwawaynetrunner** [comment]
Not only is Rec Studio revealing the hosted cards in the logs, it's also occasionally behaving very strangely with the hosted cards. It seems that something is causing each hosted card to activate/apply twice.

Example: I hosted a Tech Startup on an empty Rec Studio from HQ. Next turn, I rez Tech Startup and use it to bring out Sandburg from R&D onto Rec Studio. Despite being face-down, Sandburg increased the strength of all my Ice appropriately. Upon rezzing Sandburg, the amount was doubled (ie +14 strength instead of +7 while I was at 35 credits). I then hosted an Oaktown Renovations on the same Rec Studio (with the Sandburg), and each advancement of the Oaktown applied twice (ie +4 credits total, the log showed it popping up twice in a row per advancement).
